wasm/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ endif()
 set_target_properties(bergamot-translator-worker PROPERTIES
                         SUFFIX ".js"
                         LINK_FLAGS ${LINKER_FLAGS}
-                        )
+                        RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}
```

Comment:
Why did we require this change? This is what's breaking tests?

Reply:
Yes. Submitted the fix for it with the latest commit.